packages/expo-build-properties/src/pluginConfig.ts

```diff
@@ -130,6 +154,51 @@ export interface PluginConfigTypeAndroid {
   manifestQueries?: PluginConfigTypeAndroidQueries;
 }
 
+export interface AndroidMavenRepository {
+  /**
+   * The URL of the Maven repository.
+   */
+  url: string;
+  /**
+   * The credentials to use when accessing the Maven repository.
+   * May be of type PasswordCredentials, HttpHeaderCredentials, or AWSCredentials.
+   *
+   * @see the authentication schemes section of [Gradle documentation](https://docs.gradle.org/current/userguide/declaring_repositories.html#sec:authentication_schemes) for more information.
+   */
+  credentials?: AndroidMavenRepositoryCredentials;
+  /**
+   * The authentication scheme to use when accessing the Maven repository.
+   */
+  authentication?: AndroidMavenRepositoryAuthenticationScheme;
```

Comment:
```suggestion
  authentication?: 'basic' | 'digest' | 'header';
```

that's a very nit comment. internally we prefer not to use typescript enum. wdyt if we could replace with string union here?

Reply:
No problem here. I can understand why some prefer the string union. This change is pushed.